src/components/v2/VoteProposalUi/index.tsx

```diff
@@ -0,0 +1,225 @@
+      label: string;
+    }
+  > = useMemo(
+    () => ({
+      queued: {
+        iconWrapperCss: [styles.iconWrapper, styles.iconDotsWrapper],
+        iconName: 'dots',
+        iconCss: styles.icon,
+        label: t('voteProposalUi.statusCard.queued'),
+      },
+      readyToExecute: {
+        iconWrapperCss: [styles.iconWrapper, styles.iconInfoWrapper],
+        iconName: 'exclamation',
+        iconCss: styles.icon,
+        label: t('voteProposalUi.statusCard.readyToExecute'),
+      },
+      executed: {
+        iconWrapperCss: [styles.iconWrapper, styles.iconMarkWrapper],
+        iconName: 'mark',
+        iconCss: [styles.icon, styles.iconCheck],
+        label: t('voteProposalUi.statusCard.executed'),
+      },
+      cancelled: {
+        iconWrapperCss: [styles.iconWrapper, styles.iconCloseWrapper],
+        iconName: 'close',
+        iconCss: styles.icon,
+        label: t('voteProposalUi.statusCard.cancelled'),
+      },
+    }),
+    [],
+  );
+
+  switch (status) {
+    case 'queued':
+    case 'readyToExecute':
+    case 'executed':
+    case 'cancelled':
+      return (
+        <>
+          <div css={statusContent[status].iconWrapperCss}>
+            <Icon css={statusContent[status].iconCss} name={statusContent[status].iconName} />
+          </div>
+          <Typography css={styles.statusText} variant="body2">
+            {statusContent[status].label}
+          </Typography>
+        </>
+      );
+    default:
+      return <Spinner variant="small" />;
+  }
+};
+
+type UserVoteStatus = 'votedFor' | 'votedAgainst' | 'abstained';
+
+interface IVoteProposalUiProps {
+  className?: string;
+  proposalNumber: number;
+  proposalText: string;
+  proposalStatus: ProposalStatus;
+  cancelDate?: Date;
+  userVoteStatus?: UserVoteStatus;
+  votedForWei?: BigNumber;
+  votedAgainstWei?: BigNumber;
+  abstainWei?: BigNumber;
+  tokenId?: TokenId;
+}
+
+export const VoteProposalUi: React.FC<IVoteProposalUiProps> = ({
+  className,
+  proposalNumber,
+  proposalText,
+  proposalStatus,
+  cancelDate,
+  userVoteStatus,
+  votedForWei,
+  votedAgainstWei,
+  abstainWei,
+  tokenId,
+}) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+
+  const voteStatusText = useMemo(() => {
+    switch (userVoteStatus) {
+      case 'votedFor':
+        return t('voteProposalUi.voteStatus.votedFor');
+      case 'votedAgainst':
+        return t('voteProposalUi.voteStatus.votedAgainst');
+      case 'abstained':
+        return t('voteProposalUi.voteStatus.abstained');
+      default:
+        return t('voteProposalUi.voteStatus.notVoted');
+    }
+  }, [userVoteStatus]);
+
+  const countdownRenderer = ({
+    days,
+    hours,
+    minutes,
+    seconds,
+    completed,
+  }: CountdownRenderProps) => {
+    if (completed) {
+      // Render a completed state
+      return null;
+    }
+    // Render a countdown
+    if (days) {
+      return `${days}d ${hours}h : ${minutes}m : ${seconds}s`;
```

Comment:
We will need to internationalize this. The best way would be to create a new format 
[Example](https://github.com/VenusProtocol/venus-protocol-interface/commit/b26e35daf776f5a86ad2b69ac0bb67d69334f3ad#diff-e9a58818fe30c08eef8d1304431ed8efc18964455adcddf1f82520d6611602feR30)

[Translation](https://github.com/VenusProtocol/venus-protocol-interface/commit/b26e35daf776f5a86ad2b69ac0bb67d69334f3ad#diff-7e59a3788c0d1ce74e3a477e90c73f5ff8b7663350fb34d482313cde321e4846R152)